tests/test_psfCandidate.py

```diff
@@ -139,6 +188,48 @@ def testFaintNeighborMasking(self):
         self.checkCandidateMasking([(self.x+5, self.y, 0.5)], threshold=0.9, pixelThreshold=1.0)
 
 
+class MakePsfCandiatesTaskTest(lsst.utils.tests.TestCase):
```

Comment:
Probably don't want a typo in the class name ("Candidates") 😄

Reply:
Clearly I cannot type that word.